tests/mlmodel_langchain/_mock_external_langchain_server.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2010 New Relic, Inc.
```

Comment:
There are a bunch of changes to the mock file and the conftest in openai for openai v1 that we'll need to pull in here. Maybe let's pair on that later this week?